Automattic/wp-calypso
# Jetpack Manage: Add 'Pricing' nav menu in the sidebar.

This PR adds a new 'Pricing' menu in the Jetpack Manage sidebar, providing quick access to the Issue licenses page.

<img width="2547" alt="Screen Shot 2024-01-17 at 3 57 36 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/fcd77b10-b084-47c0-baff-0024bca7b18d">


Closes https://github.com/Automattic/jetpack-manage/issues/231

## Proposed Changes

* Add a 'Pricing' menu item linked to the Issue license page in the Sidebar component.
## Testing Instructions

Since these changes are made specifically for agencies, you must set yourself(partner) as an agency - 2c49b-pb.

* Use the Jetpack cloud live link below and Go to the Overview page (`/overview`)
* Confirm that in the sidebar, the new **'Pricing'** menu item is visible.
* Click the **'Pricing'** menu item and confirm it redirects to the Issue license page.

## Pre-merge Checklist
